package-res/resources/web/pentaho/visual/base/View.js

```diff
@@ -135,6 +141,15 @@ define([
       });
     },
 
+    /**
+     * Updates a visualization.
+     *
+     * If the visualization is valid, the visualization element will be created on the first update
+     * and proceed with the visualization update, otherwise it will be rejected and prevent the update.
```

Comment:
and then proceed with the visualization update; otherwise, it will be rejected and prevent the update.